styles.css

```diff
@@ -0,0 +1,971 @@
+	cursor: pointer;
+	max-width: 200px;
+	border: 1px solid var(--color-grey-bg);
+	display: border-box;
+	overflow: hidden;
+}
+
+.card-img-container {
+	width: 100%;
+	overflow: hidden;
+}
+
+.responsive-img {
+	width: 100%;
+	height: 100%;
+}
+
+.card-description {
+	padding: 0.15rem 0.5rem;
+	display: block;
+	flex-direction: column;
+}
+
+.brand-wishlish-wrapper {
+	display: grid;
+	grid-template-columns: 3fr 1fr;
+	align-items: center;
+	margin: 0.2rem 0rem;
+	padding: 0;
+}
+
+.product-name {
+	margin: 0px;
+	font-size: 0.9rem;
+	font-weight: bold;
+	color: var(--color-grey-darkest);
+	margin: 0.2rem 0rem;
+	/* overflow-wrap: ; */
+	white-space: nowrap;
+	overflow-wrap: break-word;
+	word-wrap: break-word;
+	/* text-overflow: ; */
+}
+
+.btn-wishlist {
+	background-color: inherit;
+	color: var(--color-grey);
+	justify-self: end;
+	padding: 0;
+	margin: 0;
+}
+
+.btn-wishlist-added {
+	background-color: inherit;
+	color: var(--color-danger);
+	justify-self: end;
+	padding: 0;
+	margin: 0;
+}
+
+.btn-cart-plus {
+	background-color: inherit;
+	color: var(--color-primary);
+	justify-self: end;
+	padding: 0;
+	margin: 0;
+	font-size: 1.2rem;
+}
+
+.product-brand {
+	font-size: 0.7rem;
+	color: var(--color-grey);
+}
+
+.price {
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+	justify-content: start;
+	font-size: 0.7rem;
+	margin-top: 0.3rem;
+}
+.discounted-price {
+	/* font-size: 0.7rem; */
+	color: var(--color-primary);
+}
+.mrp {
+	/* font-size: 0.8rem; */
+	color: var(--color-grey);
+	text-decoration: line-through;
+	text-decoration-color: var(--color-grey);
+}
+
+.stock-warning {
+	color: var(--color-warning);
+	font-size: 0.65rem;
+	margin: 0.2rem 0rem;
+}
+
+.card-badge {
+	position: absolute;
+	left: 0;
+	top: 2rem;
+	background: var(--color-secondary);
+	padding: 0.25rem;
+	color: #fff;
+	font-size: 0.75rem;
+	font-weight: bold;
+}
+
+.card-discount-badge {
+	position: absolute;
+	right: -1.1rem;
+	top: 0.9rem;
+	text-align: center;
+	width: 5rem;
+	transform: rotate(45deg);
+	background: var(--color-warning);
+	padding: 0.1rem;
+	color: #fff;
+	font-size: 0.7rem;
+	font-weight: bold;
+}
+
+/*Text Cards*/
+/* text card */
+.txt-card {
+	width: 100%;
+	padding: 1rem;
+	margin: 0.5rem;
+	box-shadow: 13px 13px 21px -5px var(--color-grey-bg);
+	border: 1px solid var(--color-grey-bg);
+	/* 6px 6px 6px 3px white; */
+}
+
+.txt-card .title {
+	font-size: 1.5rem;
+	color: var(--text-color);
+	margin: 0;
+	font-weight: 600;
+	margin-bottom: 0.4rem;
+}
+
+.txt-card .subtitle {
+	font-size: 1rem;
+	color: var(--color-grey-dark);
+	margin: 0;
+	margin-bottom: 0.7rem;
+}
+
+.txt-card .txt-content {
+	font-size: 0.7rem;
+	color: var(--color-grey-dark);
+	margin-bottom: 0.4rem;
+}
+
+/*Progress Bar*/
+.progress {
+	height: 1rem;
+	width: 100%;
+	background-color: var(--color-grey-bg);
+	margin: 0.5rem;
+}
+.progress-bar{
+	height: 1rem;
+	margin: 0;
+	background-color: var(--color-primary);
+}
+
+
+/* form */
+.form-container {
+	min-width: 40vw;
```

Comment:
I'm not sure about the viewport unit, read about it once as the viewport is different for every screen!

Reply:
Sure will do that.